src/huntsman/pocs/scheduler/observation/flat.py

```diff
@@ -0,0 +1,50 @@
+import os
+from contextlib import suppress
+
+from astropy import units as u
+from huntsman.pocs.utils import dither
+from panoptes.pocs.scheduler.field import Field
+from huntsman.pocs.scheduler.observation.dithered import DitheredObservation
+
+
+class DitheredFlatObservation(DitheredObservation):
+
+    def __init__(self, position, pattern=dither.dice9, n_positions=9, dither_offset=0.5 * u.arcmin,
+                 random_offset=0.5 * u.arcmin, *args, **kwargs):
+        """
+        Args:
+            position (str or astropy.coordinates.SkyCoord): Center of field, can
+                be anything accepted by `astropy.coordinates.SkyCoord`.
+        """
+        # Convert from SkyCoord if required
+        with suppress(AttributeError):
+            position = position.to_string('hmsdms')
+        field = Field('Flat', position=position)
+
+        super().__init__(field=field, *args, **kwargs)
+
+        # Listify the exposure time
+        self.exptime = [self.exptime for _ in range(n_positions)]
+
+        # Setup the dither fields
+        dither_coords = dither.get_dither_positions(field.coord, n_positions=n_positions,
+                                                    pattern=pattern, pattern_offset=dither_offset,
+                                                    random_offset=random_offset)
+        self.field = [Field(f'FlatDither{i:02d}', c) for i, c in enumerate(dither_coords)]
```

Comment:
I can see us getting to 100. Maybe?

```suggestion
        self.field = [Field(f'FlatDither{i:03d}', c) for i, c in enumerate(dither_coords)]
```